UndertaleModTool/Scripts/Community Scripts/ImportGMS2FontData.csx

```diff
@@ -89,17 +103,19 @@ if (fontData.ContainsKey("ascender"))
 if (fontData.ContainsKey("ascenderOffset"))
     font.AscenderOffset = (int)fontData["ascenderOffset"];
 
-font.RangeStart = 0;
-font.RangeEnd = 0;
+ushort rangeStart = 0;
+uint rangeEnd = 0;
 foreach (JObject range in fontData["ranges"].Values<JObject>())
 {
-    var rangeStart = (ushort)range["lower"];
-    var rangeEnd = (uint)range["upper"];
-    if (font.RangeStart > rangeStart)
-        font.RangeStart = rangeStart;
-    if (font.RangeEnd > rangeEnd)
-        font.RangeEnd = rangeEnd;
+    var rangeStartChk = (ushort)range["lower"];
```

Comment:
What's Chk short for?

Reply:
`Check` IIRC, I'm probably going to drop these variables though